docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

```diff
@@ -1,29 +1,23 @@
 [id="creating-a-ceph-nfs-cluster_{context}"]
 
-= Creating a NFS Ganesha cluster
+= Creating an NFS Ganesha cluster
 
-If you use the Ceph via NFS backend with {rhos_component_storage_file_first_ref}, prior to adoption, you must create a new clustered NFS service on the Ceph cluster. This service will replace the standalone, pacemaker-controlled `ceph-nfs` service that was used on {rhos_prev_long} {rhos_prev_ver}.
+If you use the Ceph via NFS back end with the {rhos_component_storage_file_first_ref}, you must create a new clustered NFS service on the {Ceph} cluster. This service replaces the standalone, Pacemaker-controlled `ceph-nfs` service that you use in {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver}.
 
 .Procedure
 
-. You must identify the ceph nodes to deploy the new clustered NFS service.This service must be deployed on the `StorageNFS` isolated network so that
-it is easier for clients to mount their existing shares through the new NFS
-export locations.
-. You must propagate the `StorageNFS` network to the target nodes
-where the `ceph-nfs` service will be deployed.
+. Identify the {Ceph} nodes to deploy the new clustered NFS service. You must deploy this service on the `StorageNFS` isolated network so that
+you can mount your existing shares through the new NFS export locations.
+. To propagate the `StorageNFS` network to the target nodes where the `ceph-nfs` service is deployed, identify the node definition file that is used in the {OpenStackShort} environment. This is the input file associated with the `openstack overcloud node provision` command. For example, this file might be called `overcloud-baremetal-deploy.yaml`.
 ifeval::["{build}" != "downstream"]
 See link:https://docs.openstack.org/project-deploy-guide/tripleo-docs/wallaby/features/network_isolation.html#deploying-the-overcloud-with-network-isolation[Deploying
 an Overcloud with Network Isolation with TripleO] and link:https://docs.openstack.org/project-deploy-guide/tripleo-docs/wallaby/post_deployment/updating_network_configuration_post_deployment.html[Applying
 network configuration changes after deployment] for the background to these
 tasks.
 endif::[]
-The following steps will be relevant if the Ceph Storage nodes were
-deployed via {OpenStackPreviousInstaller}.
-.. Identify the node definition file used in the environment. This is
-the input file associated with the `openstack overcloud node provision`
-command. For example, this file may be called `overcloud-baremetal-deploy.yaml`
-.. Edit the networks associated with the {CephCluster} nodes to include the
-`StorageNFS` network:
+The following steps are relevant if you deployed your {Ceph} nodes with {OpenStackPreviousInstaller}.
```

Comment:
Is this whole procedure relevant for Ceph Storage nodes that were deployed via director? Or only these steps, and then the rest of the procedure is relevant to both director-deployed Ceph and external Ceph?

Reply:
And do we need to be any more specific about how to identify the node definition file?

Reply:
@fmount I'm not sure if you saw these 2 comments above. ^

Reply:
Sorry, I missed this comment (the GH interface doesn't help me when there are a lot of comments). Replying ....

Reply:
Either if the Ceph is director deployed or external, we used to deploy Ganesha with TripleO. Now this pattern changes in 18, and we ask, as part of this guide, to redeploy ganesha (aka ceph-nfs cluster) in the nodes that are part of Ceph, and that are not going to be decommissioned.
This means that 90 % of this procedure applies to both use cases, except for this part related to TripleO, that doesn't apply to a cluster that is externally managed.
However, we might want to put a note about external Ceph, saying that in that case, the human administrator should take care of:
1. choosing the network where the ceph-nfs should be exposed (in terms of VIP/frontend network)
2. select the VIP that will be used for Ganesha when the `ceph nfs create ...` command is run.

@klgill do we want to make this distinction here, at the very beginning of the procedure, and put a Note related to the external Ceph cluster scenario?

Reply:
@fmount OK, will need to change step 2 back to its original format, which was that the director-deployed steps were substeps under step 2. And I can write "If you are using director-deployed Ceph, propogate the StorageNFS network..." Then maybe have a note about external Ceph below step 2.

Who is doing the director-deployed steps, if not a human administrator?

Reply:
ok thanks, and yes, a human administrator in both cases, it's just that it might be not the same person when Ceph is external (Ceph cluster admin != openstack admin), but this is a minor thing and up to the customer, it's human in any case :D

Reply:
@fmount I formatted step 2 as substeps again, and introduced the step with, "If you deployed your Red Hat Ceph Storage nodes with director..."

Can you also review steps 4-7? Those are the commands that I converted to steps. 
After I get your approval, I'll send this PR to peer review.